tests/source-selection.spec.ts

```diff
@@ -59,17 +62,21 @@ test('Source selection in in-game menu', async ({ page, context, browser }) => {
   await pages.advancedConnectionPage.saveAndGoToSing();
   await pages.mainMenuPage.goToSingSong();
   await pages.songLanguagesPage.continueAndGoToSongList();
-  await pages.songListPage.openPreviewForSong(song1);
-  await pages.songPreviewPage.goNext();
-  await pages.songPreviewPage.playTheSong();
-  await pages.gamePage.waitForPlayersScoreToBeGreaterThan(100);
-  await page.keyboard.press('Backspace');
 
-  // Make sure the input isn't monitored anymore if it's not in use
-  await pages.gamePage.microphonesSettings();
-  await pages.advancedConnectionPage.togglePlayerMicrophoneSource(blueMicNum);
-  await pages.advancedConnectionPage.saveAndGoToSing();
-  await expectMonitoringToBeEnabled(page);
+  await test.step('Play the song and wait for players greater-than-expected score', async () => {
```

Comment:
just `play the song` (`waitForPlayersScoreToBeGreaterThan` is meant to make sure the game actually started)